pages/index.html

```diff
@@ -0,0 +1,59 @@
+---
+layout: homepage
+title: CBA is an extension for chrome browser automation
+menu: Homepage
+---
+
+<img src="/images/Manvel_440x280_border.jpg" alt="CBA-banner" />
+<ol>
+  <li>
+    CBA gives ability to automate chrome browser.
+  </li>
+  <li>
+    Ability to setup step-by-step code injections from Extensions popup
+    interface.
+  </li>
+  <li>
+    Ability to Record interactions in the chrome browser and playing them.
+  </li>
+  <li>
+    Projects saving in browser&#39;s local storage.
+  </li>
+  <li>
+    Step by step code injection.
+  </li>
+  <li>
+    Ability of projects import and export.
+  </li>
+  <li>
+    Ability of using CBA ready functions.
+  </li>
+  <li>
+    Ability to write your own code that will have access to web page&#39;sDOM.
+  </li>
+  <li>
+    And much much more...
+  </li>
+</ol>
+
+<section class="box">
+  <h3>Install</h3>
+  <p>
+    <img src="/images/arrow.png" />
+    Get chromium browser automation from google web store.
+  </p>
+  <a
+    href="https://chrome.google.com/webstore/detail/jmbmjnojfkcohdpkpjmeeijckfbebbon"
+    target="_blank"
+    >CBA in chrome web store</a
+  >
+</section>
+
+<section class="box">
+  <h3>Documentation</h3>
+  <p>
+    <img src="/images/doc_grey50.png" />
+    Learn more about how CBA works.
+  </p>
+  <a href="/documentation">Documentation</a>
+</section>
```

Comment:
I was thinking about flex implementation, but there must be fixed width of the row for flex-wrap property to function which brings to difficulties with calculations of actual width. That would be more obvious way of styling but again it will depend on max-width value thus will be risky. I think of current implementation not very obvious, but easy to manipulate when it comes to adding new grid items. `grid-template-columns` and `grid-column` can in specific cases interfere with each other but it will be easy to take care of conflict considering that that `grid-column` (or `grid-column-start` and `grid-column-end`) is for positioning items in grid, `grid-template-columns` is for defining grid template. Anyway, I understand your concerns about fragility (because the properties can interfere with each other, but not at current values) but like this kind of implementation for it's semantic simpleness.

Reply:
> but there must be fixed width of the row for flex-wrap property 

Not sure if I understand this, but whole point flexbox is that you don't need fixed width.

> but it's easy to manipulate when it comes to adding new grid items

Easy for you, but not for content writers. Either you provide content writers with obvious layout management classes, or limit content they can touch.

I was more thinking that we might be able to move most of the implementation details out of the `page/index.html`. Maybe:
```
index.md
---
title: Homepage
menu: Homepage
layout: homepage
hero-image: /images/Manvel_440x280_border.jpg
box-download-title: Download
box-download-content: Get chromium browser automation from google web store.
box-download-button: CBA in chrome webstore
...
---

1. CBA gives ability to automate chrome browser.
1. Ability to setup step-by-step code injections from Extensions popup interface.
1. Ability to Record interactions in the chrome browser and playing them.
...
```

But even for this implementation text content needs to be wrapped and all the effort to avoid them will gone.

Or I was thinking that maybe we can provide some self explanatory styling options, so the whole page will be less error prone to edit.

Reply:
To be followed up in -> https://github.com/browser-automation/cba-website/issues/30